vite.config.js

```
@@ -1,20 +1,50 @@
+        // login: 'src/login-main.ts',
+        // account: 'src/account-main.ts',
+        // For now, stick to a single 'main' input.
       },
+      output: [
+        {
+          dir: 'themes/modern-theme/login/resources',
+          format: 'es', // Or 'iife', 'umd' depending on Keycloak's JS needs in FTLs
+          entryFileNames: 'js/main.js',
+          assetFileNames: (assetInfo) => {
+            // Ensure assetInfo.name exists before calling endsWith
+            if (assetInfo.name && assetInfo.name.endsWith('.css')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)